go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -254,6 +255,14 @@ var metricIPsecEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "Specifies whether IPSec is enabled for this cluster(1) or not enabled for this cluster(0)",
 })
 
+var metricIPsecTunnelState GaugeVecInterface = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: types.MetricOvnkubeNamespace,
+	Subsystem: types.MetricOvnkubeSubsystemController,
+	Name:      "ipsec_tunnel_state",
+	Help:      "Specifies whether IPSec Child SA is established for the Geneve tunnel(1) or not(0)"},
+	[]string{"remote_ip"},
```

Comment:
unbounded cardinality :( we cannot downstream this commit.

Reply:
the intention is have a metric for every geneve tunnel, so what is your thinking ? just to have a single metric which conveys overall tunnel status ?

Reply:
ya, unless you can safely say the cardinality is < 14.